server/src/main/java/io/deephaven/server/table/ops/UpdateByGrpcImpl.java

```diff
@@ -349,7 +350,10 @@ private static WindowScale adaptWindowScale(UpdateByWindowScale timescale) {
             case TICKS:
                 return WindowScale.ofTicks(timescale.getTicks().getTicks());
             case TIME:
-                return WindowScale.ofTime(timescale.getTime().getColumn(), timescale.getTime().getPeriodNanos());
+                return timescale.getTime().hasDurationString()
```

Comment:
Do there need to be any checks to see if both the integer and string values have been set, or does setting one eliminate the other?

Reply:
By specifying `oneof`, the `.proto` definition eliminates any chance of setting both.  Per the documentation:
```
Setting any member of the oneof automatically clears all the other members.
```